src/pyobo/registries/obofoundry.json

```diff
@@ -6137,4 +6137,4 @@
     },
     "title": "Maize gross anatomy"
   }
-]
\ No newline at end of file
+]
```

Comment:
This file gets periodically re-written by python, so we'll maintain a lack of trailing new lines (i know, this makes me upset too)

Reply:
Okay so in the last update (085afae) I reorganized the registry code so it's a bit more clear how to update these.

However, the registries can easily be wrong. After updating OBO Foundry, this was not fixed. For this, we have the metaregistry.json. In each entry, if there's an alternate OBO URL, you can add the "download" key. I guess I should document what's in the metaregistry in case anyone goes to read it

Reply:
Documentation about metaregistry added to README.rst in ac9a624

Reply:
Now that I made the update to the metaregistry in https://github.com/pyobo/pyobo/pull/38/commits/90b66646622faa614a49c990118f57f6e5ca5a16, we can take this change away